SU2_CFD/src/output_structure.cpp

```diff
@@ -10394,30 +10394,50 @@ void COutput::LoadLocalData_Flow(CConfig *config, CGeometry *geometry, CSolver *
     
     nVar_Par += 3;
     Variable_Names.push_back("Temperature");
-    Variable_Names.push_back("Pressure_Coefficient");
+		if (config->GetOutput_FileFormat() == PARAVIEW){
```

Comment:
Thanks for adding these. Once the new restarts are in, let's check that this works with ParaView.